Documentation/user-guides/cluster-monitoring.md

```diff
@@ -60,7 +60,7 @@ spec:
       - args:
         - --kubelet-service=kube-system/kubelet
         - --config-reloader-image=quay.io/coreos/configmap-reload:v0.0.1
-        image: quay.io/coreos/prometheus-operator:v0.18.1
+        image: quay.io/coreos/prometheus-operator:0.19.0
```

Comment:
Preceding `v` will be added once https://github.com/coreos/prometheus-operator/pull/1254 is merged.